exporters/otlp-http/trace/build.gradle.kts

```diff
@@ -13,6 +13,7 @@ dependencies {
 
   implementation(project(":exporters:otlp:common"))
 
+  implementation("com.google.api.grpc:proto-google-common-protos")
```

Comment:
Needing a dependency on a gRPC proto in the simple HTTP exporter seems like a bug in the spec to me

Reply:
ooh. interesting. yeah, I had forgotten this was the case. And, I agree with you, although I bet the otlp proto folks will fight hard not to have to re-implement the basic stuff (happy to be wrong).